src/Lightning.ts

```diff
@@ -290,6 +291,26 @@ export const LightningMixin = (superclass) =>
         params,
       })
 
+      const remainingTwoFactorLimit = await this.user.remainingTwoFactorLimit()
+
+      if (
```

Comment:
this logic of looking at if twoFactor is enabled it not present for onchain. is that intended?

Reply:
Nope, not at all. I missed that. I've added it, and also a test for it.